geonode_mapstore_client/client/js/components/FiltersForm/FilterByExtent.jsx

```diff
@@ -123,22 +128,21 @@ function FilterByExtent({
                                 type: 'vector',
                                 features: [getFeatureFromExtent(queryExtent)],
                                 style: vectorLayerStyle
-                                    ? vectorLayerStyle
+                                    ? {...vectorLayerStyle, stroke: isExtentInIDL && {width: 0, color: '#397AAB'}}
```

Comment:
I would not prefer to have always the same style. So let's remove the stroke from the rendered box. I noticed that width or weight equal to 0 has no effect probably because it's interpreted as undefined. I would suggeste to use a small value for weight so the stroke is not visible anymore.

```js
{ ...vectorLayerStyle, weight: 0.001 }
```